pkg/kubeadm/master/manifests.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubemaster
+
+import (
+	"bytes"
+	"encoding/json"
+	_ "fmt"
+	"os"
+	"path"
+
+	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	api "k8s.io/kubernetes/pkg/api/v1"
+	kubeadmapi "k8s.io/kubernetes/pkg/kubeadm/api"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util/intstr"
+)
+
+// Static pod definitions in golang form are included below so that `kubeadm
```

Comment:
Ah, ok. We already respect `KUBE_PREFIX_ALL`, so that's doable... however, in the context of the general situation of us writing a handful of files that don't have an obvious connection in isolation, I was thinking that tarball export may be not a bad idea for exactly that reason, which would be easy once we collect all the files in some kind of abstraction (https://github.com/kubernetes/kubernetes/pull/31221#discussion_r76956119)...